flang/test/Lower/OpenMP/omp-parallel-private-clause.f90

```diff
@@ -41,3 +41,22 @@ subroutine private_clause(arg1, arg2, arg3, arg4)
 !$OMP END PARALLEL
 
 end subroutine
+
+!FIRDialect: func @_QPmultiple_private_fix(%{{.*}}: !fir.ref<i32>)
+!FIRDialect-DAG: %{{.*}} = fir.alloca i32 {{{.*}}uniq_name = "{{.*}}j"}
+!FIRDialect-DAG: %{{.*}} = fir.alloca i32 {{{.*}}uniq_name = "{{.*}}x"}
+!FIRDialect-DAG: omp.parallel
+!FIRDialect-DAG: %{{.*}} = fir.alloca i32 {{{.*}}uniq_name = "{{.*}}j"}
+!FIRDialect-DAG: %{{.*}} = fir.alloca i32 {{{.*}}uniq_name = "{{.*}}x"}
+!FIRDialect-DAG: omp.wsloop
```

Comment:
Would using `mlir/utils/generate-test/checks.py` work here for tighter checking?

Reply:
OK. I have now use the script to generate checks.